pkg/kosmosctl/rsmigrate/options.go

```diff
@@ -0,0 +1,71 @@
+package rsmigrate
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+	"k8s.io/client-go/kubernetes"
+	restclient "k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	ctlutil "k8s.io/kubectl/pkg/cmd/util"
+
+	"github.com/kosmos.io/kosmos/pkg/apis/kosmos/v1alpha1"
+	kosmosversioned "github.com/kosmos.io/kosmos/pkg/generated/clientset/versioned"
+)
+
+type KnodeOptions struct {
+	KnodeName         string
+	Knode             *v1alpha1.Knode
+	KnodeNativeClient kubernetes.Interface
+	KnodeCrdClient    *kosmosversioned.Clientset
+}
+
+type CommandOptions struct {
+	MasterKubeConfig string
+	MasterClient     kubernetes.Interface
+	MasterCrdClient  *kosmosversioned.Clientset //clientset operate knode releted resource
```

Comment:
Comments are typically placed above the line of code.

Reply:
modified